string station_id                 # Nombre de la estación
geometry_msgs/Point coordinates   # Coordenadas promedio de la estación
geometry_msgs/Point[] polygon     # Array de puntos que delimitan el polígono de la estación
